docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -163,69 +156,55 @@ def fetch_latest_version(version)
         end
       end
 
-      def comparable_tags_from_registry(version)
-        original_prefix = prefix_of(version)
-        original_suffix = suffix_of(version)
-        original_format = format_of(version)
+      def comparable_tags_from_registry(original_tag)
+        original_prefix = original_tag.prefix
+        original_suffix = original_tag.suffix
+        original_format = original_tag.format
```

Comment:
These three intermediate vars look like they could all be deleted and actually improve readability through conciseness?

Reply:
Same as with the other comment, I can do this as a follow up to not add more complexity to this PR.